fedimint-client/src/module/init/recovery.rs

```diff
@@ -405,7 +405,7 @@ where
             self.api().clone(),
             *self.core_api_version(),
             client_ctx.decoders(),
-            block_stream_session_range,
+            common_state.next_session..common_state.end_session,
```

Comment:
unrelated change snuck in

Reply:
Oh, I accidentally inlined `block_stream_session_range` here. No biggy.